packages/tasks/src/model-libraries.ts

```diff
@@ -242,6 +242,16 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		docsUrl: "https://huggingface.co/docs/hub/keras",
 		snippets: snippets.keras,
 		filter: true,
+		countDownloads: `path:"config.json" OR path_extension:"keras"`,
+	},
+	"tf-keras": {
+		// Legacy "Keras 2" library (tensorflow-only)
+		prettyLabel: "TF-Keras",
+		repoName: "TF-Keras",
+		repoUrl: "https://github.com/keras-team/tf-keras",
+		docsUrl: "https://huggingface.co/docs/hub/tf-keras",
+		snippets: snippets.tf_keras,
+		filter: true,
```

Comment:
Should we still offer both for filtering?

Reply:
We discussed it offline with @julien-c and agreed it would make sense at least for a transition period. `tf-keras` icon is grey-ed out to clearly show it's deprecated.